EFCore.SqlServer.HierarchyId.Test/NullabilityTests.cs

```diff
@@ -0,0 +1,287 @@
+﻿using Microsoft.EntityFrameworkCore.Test.Operators;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.EntityFrameworkCore.SqlServer
+{
+    public class NullabilityTests
+    {
+        [Fact]
+        public void Null_equalTo_null_isTrue()
+        {
+            const Operator @operator = Operator.Equal;
+
+            var vals = ((HierarchyId)null, (HierarchyId)null);
+            Assert.True(@operator.Execute(vals), @operator.GetFailureMessage(vals));
```

Comment:
You don't need operator (or even failure message part).
Just test `Assert.True(((HierarchyId)null) == ((HierarchyId)null));`

Likewise for other operators.
You can also combine like this 
```C#
var left = (HierarchyId)null;
var right = (HierarchyId)null;

Assert.True(left == right);
Assert.False(left != right);
Assert.False(left > right); //and so on for rest
```

Reply:
ok i'll update those and get rid of the overhead.